### PR TITLE
[Bz 1997080]: fixes docker-to-podman issue in tier1

### PR DESCRIPTION
**Pull request by Ameenasuhani**
Signed-off-by: ameenasuhani <ameenasuhani@gmail.com>

# Description
The add-mon.yml playbook only targets the mons and osds nodes and doesn't included the dashboard bits.
So adding a new monitor via this playbook won't deploy node-exporter on that node.
You can achieve the same by re-running the main playbook with --limit mons option (like you did for osds/rgws).

Bz: https://bugzilla.redhat.com/show_bug.cgi?id=1997080
Doc bz:https://bugzilla.redhat.com/show_bug.cgi?id=2002293

Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-UVK46F/config_roll_over_mon_daemon_0.log

### CODE REVIEW

File: tests/ceph_ansible/test_ansible_roll_over.py [hunk 1]
```diff
@@ -145,6 +145,13 @@ def run(ceph_cluster, **kw):
     )
     out, rc = ceph_installer.exec_command(cmd=cmd, long_running=True)
 
+    if "4.2" in build:
```
Comment (sunilkumarn417) [inline tests/ceph_ansible/test_ansible_roll_over.py:148]: can we have this statement,
```
if build == "4.2":
```

File: tests/ceph_ansible/test_ansible_roll_over.py [hunk 2]
```diff
@@ -145,6 +145,13 @@ def run(ceph_cluster, **kw):
...
+        if daemon == "mon":
+            cmd = "cd {} ; ANSIBLE_STDOUT_CALLBACK=debug;ansible-playbook -vvvv -i hosts {} --limit mons".format(
```
Comment (sunilkumarn417) [inline tests/ceph_ansible/test_ansible_roll_over.py:150]: can we use f-strings,
```
cmd = f"cd {ansible_dir} ; ANSIBLE_STDOUT_CALLBACK=debug;ansible-playbook -vvvv -i hosts {yaml} --limit mons" ```

**Review by sunilkumarn417 — COMMENTED** [review]
Hi @Ameenasuhani ,

Changes looks good. but few minor changes would be required.

can we have complete test suite run which also executed docker-to-podman playbook   successfully.

-Thanks

File: tests/ceph_ansible/test_ansible_roll_over.py (hunk 1 above)
Comment (Ameenasuhani) [reply tests/ceph_ansible/test_ansible_roll_over.py:148]: build will have 4.2-rhel-7 or 4.2-rhel-8 so this condition will fail

**Comment by Ameenasuhani** [comment]
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-34S52P/

File: tests/ceph_ansible/test_ansible_roll_over.py (hunk 1 above)
Comment (psathyan) [reply tests/ceph_ansible/test_ansible_roll_over.py:135]: We could have `if build.startswith("4.2")` not that the current implementation wouldn't work. I prefer not to change it to `build == '4.2'` now.

**Review by psathyan — APPROVED** [review]
Looks good to me

**Review by sunilkumarn417 — APPROVED** [review]
lgtm

File: tests/ceph_ansible/test_ansible_roll_over.py (hunk 1 above)
Comment (sunilkumarn417) [reply tests/ceph_ansible/test_ansible_roll_over.py:135]: > build will have 4.2-rhel-7 or 4.2-rhel-8 so this condition will fail

Sorry, Just had pipeline v2.0 in mind. 
Make sense to hold back to current logic.

**Pull request merged**